### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kn0wmad**
Added missing punctuation

### CODE REVIEW

**Review by dr-bonez — APPROVED** [review]

**Review by kn0wmad — COMMENTED** [review]
lgtm

File: backend/README.md [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
 - Scripts (run withing the `./backend` directory)
     - `build-prod.sh` - compiles a release build of the artifacts for running on ARM64
     - `build-dev.sh` - compiles a development build of the artifacts for running on ARM64
+- A Linux computer or VM
+
+## Structure
+
+The EmbassyOS backend is broken up into 4 different binaries:
+
+- embassyd: This is the main workhorse of EmbassyOS any new functionality you want will likely go here
```
Comment (chrisguida) [inline backend/README.md:16]: `embassyd: This is the main workhorse of EmbassyOS - any new functionality you want will likely go here`

File: backend/README.md [hunk 2]
```diff
@@ -1,11 +1,33 @@
...
+## Structure
+
+The EmbassyOS backend is broken up into 4 different binaries:
+
+- embassyd: This is the main workhorse of EmbassyOS - any new functionality you want will likely go here
+- embassy-init: This is the component responsible for allowing you to set up your device
+- embassy-cli: This is a CLI tool that will allow you to issue commands to embassyd and control it similarly to the UI
+- embassy-sdk: This is a CLI tool that aids in building and packaging services you wish to deploy to the Embassy
+
+Finally there is a library that supports all four of these tools.
```
Comment (elvece) [inline backend/README.md:22]: Remove this line?

Comment (kn0wmad) [reply backend/README.md:22]: Will this be in reference to the Rust Crate?  Is that published?

Comment (ProofOfKeags) [reply backend/README.md:22]: it is the `embassy` lib

**Review by elvece — DISMISSED** [review]

File: backend/README.md [hunk 3]
```diff
@@ -1,11 +1,35 @@
...
+  - `build-prod.sh` - compiles a release build of the artifacts for running on ARM64
+  - `build-dev.sh` - compiles a development build of the artifacts for running on ARM64
+- A Linux computer or VM
+
+## Structure
+
+The EmbassyOS backend is broken up into 4 different binaries:
+
+- embassyd: This is the main workhorse of EmbassyOS - any new functionality you want will likely go here
+- embassy-init: This is the component responsible for allowing you to set up your device
```
Comment (dr-bonez) [inline backend/README.md:18]:
```suggestion
- embassy-init: This is the component responsible for allowing you to set up your device, and handles system initialization on startup
```

**Review by dr-bonez — APPROVED** [review]

**Review by dr-bonez — APPROVED** [review]

**Pull request merged**